UnboundLib/Utils/UI/Links.cs

```diff
@@ -8,37 +8,34 @@
 
 namespace UnboundLib.Utils.UI
 {
-
     public static class MainMenuLinks
     {
-
-        private static GameObject _Links = null;
-
+        private static GameObject links;
```

Comment:
The underscore prefix indicates that this is a backing field that should not be directly modified (outside of its associated property), even within this class. Casing change makes sense, but recommend adding back the underscore